newrelic/metric.go

```diff
@@ -185,6 +183,10 @@ func (m *Metric) convertTimeSeries(ctx context.Context, data responseData, value
 			log.Errorf(ctx, "unable to parse from time %v\n", slice.To)
 			continue
 		}
+		if toTime.Before(startTime) {
+			log.Infof(ctx, "point %v is before specified start time of %v, sometimes New Relic does that.\n", toTime, startTime)
```

Comment:
Not sure whether this needs to be logged every time it happens, but if you decide to log, I'd suggest to log metric name as well. And perhaps `log.Debugf` might be more appropriate here.